dashboard/src/modules/components/ComparisonComponent/common-components.jsx

```diff
@@ -67,3 +70,35 @@ export const SearchByName = () => {
     />
   );
 };
+
+export const ViewOptions = (props) => {
+  const dispatch = useDispatch();
+  const [isOpen, setIsOpen] = React.useState(false);
+  const selected = useSelector((state) => state.comparison.datasetType);
+  const onSelect = (_event, value) => {
+    console.log("selected", value);
+    dispatch(setDatasetType(value, props.currPage));
+    setIsOpen(false);
+  };
+  const onToggle = () => {
+    setIsOpen(!isOpen);
+  };
+  const options = [
+    <SelectOption key={1} value="All Datasets" />,
+    <SelectOption key={2} value="My Datasets" />,
+    <SelectOption key={3} value="Public Datasets" />,
+  ];
```

Comment:
1. "All Datasets" translates to `/api/v1/datasets` without any filter query parameters -- this is really "all datasets I'm allowed to see", but "All Datasets" is a better name 😁 
2. "My Datasets" (which is only meaningful when logged in) translates to `/api/v1/datasets?mine`. This option could be omitted if not logged in, or maybe grayed out, or even redirect to login? (Remember that you'll get a `401` error from the API if you use this without a valid authentication token.)
3. "Public Datasets" translates to `/api/v1/datasets?access=public`. The filter is redundant if you're not logged in because that's all you can see anyway. It doesn't hurt, but it's functionally identical to "All Datasets" (that is, omitting the `?access=public`).

It seems that the intent is that `<ViewOptions>` is only rendered when logged in. That's OK, since there's no need for any selection if you're not logged in: all you can see are public datasets. But would it make more sense to do the conditional rendering inside this component rather than outside, to be sure that it's consistent across the various pages that might use it?

I could also see it rendering with a message something like "only display public datasets because you're not logged in" (note: if you want to go that way, I think we should "wordsmith" that a bit!) possibly even with a link to the login page if we can redirect back to here on login. (Which is likely too much for this PR.)